### DISCUSSION TITLE
Facing new error in Deploy/dev/Kubernetes/nginx-private-ingress Please help

### DISCUSSION THREAD

**Discussion by dikshauat** [Sunbird-Ed/Community#159] [Sunbird ED configuration discussion] [answered]
@keshavprasadms
@sowmya-dixit @anandp504
@shounakacharya

TASK [helm-deploy : Create the token pubic key file for ML Services] ***********
skipping: [localhost]

TASK [helm-deploy : helm install] **********************************************
fatal: [localhost]: FAILED! => {    "changed": true,    "cmd": "helm upgrade --install --atomic --timeout 10m nginx-private-ingress /var/lib/jenkins/workspace/Deploy/dev/Kubernetes/nginx-private-ingress/kubernetes/helm_charts/core/nginx-private-ingress -n dev",    "delta": "0:00:00.069167",    "end": "2022-09-14 14:08:16.289753",    "rc": 1,    "start": "2022-09-14 14:08:16.220586"}STDERR:Error: UPGRADE FAILED: "nginx-private-ingress" has no deployed releasesMSG:non-zero return code

PLAY RECAP *********************************************************************
localhost                  : ok=3    changed=2    unreachable=0    failed=1    skipped=4    rescued=0    ignored=0   

[Pipeline] }
[Pipeline] // stage
[Pipeline] }
[Pipeline] // ansiColor
[Pipeline] }
[Pipeline] // stage
[Pipeline] ansiColor
[Pipeline] {
[Pipeline] stage
[Pipeline] { (slack_notify)
[Pipeline] sh
+ git rev-parse --short HEAD
[Pipeline] sh
+ cut -d / -f1
+ rev
+ git name-rev --name-only HEAD
+ rev
[Pipeline] sh
+ echo Deploy/dev/Kubernetes/nginx-private-ingress
[Pipeline] echo
Could not find env specific Slack channel. Check for global slack channel..[Pipeline] echo
Could not find slack environment variable. Skipping slack notification..[Pipeline] }
[Pipeline] // stage
[Pipeline] }
[Pipeline] // ansiColor
[Pipeline] ansiColor
[Pipeline] {
[Pipeline] stage
[Pipeline] { (email_notify)
[Pipeline] sh
+ echo Deploy/dev/Kubernetes/nginx-private-ingress
[Pipeline] echo
Could not find env specific email group. Check for global email group..[Pipeline] echo
Could not find global email group variable. Skipping email notification..[Pipeline] }
[Pipeline] // stage
[Pipeline] }
[Pipeline] // ansiColor
[Pipeline] }
[Pipeline] // node
[Pipeline] End of Pipeline
ERROR: script returned exit code 2
Finished: FAILURE

**Comment by beepdot** [comment, marked as answer]
Hello there. Please deploy the job, when the job is running the `helm install` section, you will be able to see nginx private ingress pods created in Kubernetes. Please see and post the logs. You will have 10 minutes to capture these logs after which helm will roll back the deployment and remove the pods.
